src/modules/feed/components/Transaction.js

```diff
@@ -20,6 +21,12 @@ export class Transaction extends Component {
     this.state = {
       imageLoading: true
     };
+    if (this.props.transaction["receiver"] != undefined && this.props.transaction["sender"] != undefined) {
```

Comment:
This is not wrong but you can use `this.props.transaction.receiver` here (without string notation). String notation is generally used to acces a property with special characters. 
Also checking for undefined is not wrong here, though you can abbreviate it when checking if a value is not `null` or `undefined` with:
``` if (this.props.transaction.receiver)```